launch/sim.launch

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0"?>
 <launch>
+	<param name="robot_description" textfile="$(find emc_simulator)/urdf/rosbot.urdf"/>
 	<rosparam command="load" file="$(find emc_simulator)/config/simbot_config.yaml" />
+	<node pkg="robot_state_publisher" type="robot_state_publisher" name="rob_st_pub" />
 	<node pkg="emc_simulator" type="pico_simulator" name="simulator"/>
+	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find emc_simulator)/rviz/config.rviz" />
```

Comment:
I would run rviz separately.

Reply:
Don't fully agree,, see your point , but given that we want the use of the simulator as easy as possible for the students, we need a really good reason to add and extra step to launching the simulator (and thus the rviz visualisation of the simulator) , since this step will become a new point of failure

Reply:
As the visualization should be the same for the real robot and the simulator. On the robot you need to run it separately, as it needs to run on your machine and the rest of the software is probably running on the robot. So you need to have a separate command for it, So when the command is already there, also use if for the simulation.